src/components/Search/SearchResultsList.tsx

```diff
@@ -14,7 +14,9 @@ interface SearchResultsListProps {
 
 function SearchResultsList(props: SearchResultsListProps) {
   const { view, mode, searchType } = props;
-  const searchState = useAppSelector((state) => state.search[searchType]);
+  const searchState = useAppSelector(
+    (state) => state.searchCompareWithBase[searchType],
+  );
   const { searchResults } = searchState;
```

Comment:
We need to be more clever. Currently this doesn't work properly, because the "new" entry in "compare with base" share the results with the "compare over time" widget. See for example this:

1. Open the dropdown for the "new" entry in "compare with base". Notice the entries.
2. Open the dropdown for the entry in "compare over time" => same entries
3. Change the repository in one of them, then open the 2 dropdowns again => entries changed in both of them

Instead, we should probably store the results of the fetch in a separate slice, that would be reused between all of the components. The results would be stored with a key for "repository".
Then from the components we can have a selector that would get the fetched value depending on the current selected repository, that should be straightforward.

I believe that it would be good to do this change in a separate PR before doing this one.

cc @Carla-Moz, this is something we discussed yesterday